src/Microsoft.Identity.Client/Internal/Requests/BaseRequest.cs

```diff
@@ -118,6 +120,10 @@ protected void ValidateScopeInput(SortedSet<string> scopesToValidate)
                 RequestContext.Logger.Error(ex);
```

Comment:
Yes, this was my bad, hence removed the comment.  Did not notice that GitHub has added compressed lines 108-118